src/desktop/apps/isomorphic-relay-example/routes.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import { graphql } from 'react-relay'
+import { Layout } from './components/Layout'
+import { HomeRoute } from './routes/home/HomeRoute'
+import { ArtistRoute } from './routes/artist/ArtistRoute'
+import { AuctionRoute } from './routes/auction/AuctionRoute'
+import { ReactLoadableClientRoute } from './routes/react-loadable/ReactLoadableClientRoute'
+import { ReactLoadableServerRoute } from './routes/react-loadable/ReactLoadableServerRoute'
+
+export const routes = [
+  {
+    Component: Layout,
+    path: '/isomorphic-relay-example',
+    children: [
+      {
+        path: '/',
+        Component: HomeRoute,
+      },
+      {
+        path: '/about',
+        Component: () => <div>About page!</div>,
+      },
+      {
+        path: '/artsy/:artistID',
+        prepareVariables: params => ({ artistID: 'pablo-picasso' }),
+        getComponent: ({ artworks }) => {
+          return import('@artsy/reaction/dist/Components/ArtworkGrid').then(
+            ({ default: ArtworkGrid }) => props => (
+              <div>
+                <ArtworkGrid {...props.artist} />
+              </div>
+            )
+          )
+        },
+        query: graphql`
+          query routes_ArtsyQuery($artistID: String!) {
+            artist(id: $artistID) {
+              artworks: artworks_connection(first: 10) {
+                ...ArtworkGrid_artworks
+              }
+            }
+          }
+        `,
```

Comment:
Ahhh, now I get what you were saying about the lack of QueryRenderers all over the place. Nice 👌